### DISCUSSION TITLE
I think I bricked my v2

### DISCUSSION THREAD

**Discussion by ahamill85** [gtxaspec/wz_mini_hacks#435] [Q&A]
I tried installing on a v2 last night and I got the audio cue that wz_mini was being installed, but then it would go in to a boot loop. I've tried reinstalling several times, but now I can't even get the camera to boot up with or without a SD card inserted. I've also tried flashing with stock firmware, which also doesn't work. Every time I boot up the camera, I get a solid yellow light for around 6 seconds, camera clicks, and then another few seconds after the light goes off. Nothing after that.

**Comment by bjs-pdx** [comment]
Have you tried factory reset?  With the camera powered on and no SD Card press and hold the setup button until the camera light goes off/changes.  Let it boot then try to install Wyze firmware.
